swc.config.ts

```diff
@@ -0,0 +1,102 @@
+/* eslint-disable @typescript-eslint/no-require-imports */
+const { swcDir } = require("@swc/cli");
```

Comment:
This must be the unusual enforced require usage you were talking about, crazy

Reply:
yep, can't use `import` because it explodes with the most bizarre errors